gobblin-compaction/src/main/java/org/apache/gobblin/compaction/CliOptions.java

```diff
@@ -80,6 +81,12 @@ public static Properties parseArgs(Class<?> caller, String[] args, Configuration
         jobConfig = JobConfigurationUtils.fileToProperties(cmd.getOptionValue(jobConfigLocation));
       } else {
         jobConfig = JobConfigurationUtils.fileToProperties(cmd.getOptionValue(jobConfigLocation), conf);
+        for (Iterator<String> it = jobConfig.stringPropertyNames().iterator(); it.hasNext();){
```

Comment:
Can you update to the following form that's a little bit cleaner?
```for (String configKey : jobConfig.stringPropertyNames()) {
                    if (conf.get(configKey) != null) {
                        conf.unset(configKey);
                      }
                  }```

Reply:
@htran1 Sure will update. Thank you.

Reply:
@htran1 @abti Have updated.